### PR TITLE
SomethingSimpl dev

### PR DESCRIPTION
**Pull request by SomethingSimpl**
added repair object to locations, added action to repair, changed Special to Radio category in buildUI

### CODE REVIEW

**Comment by SomethingSimpl** [comment]
let me know if I should prepare persistence for Fuel and Ammo as well and only comment it out or leave it be for now as bots dont do rearms/refuels at the moment.

**Comment by SomethingSimpl** [comment]
had to add our own randomization over the 3cb/vanilla solutions as it kept overriding my saved values after I loaded my texture as the init took some time (mainly 3CB). added benefit is all clients will see same textures on vehicles. this way all civilian vehicles are random, all military are exactly as defined by the template and textures persist across saves and respawns correctly.

File: src/Unit/Unit.sqf [hunk 1]
```diff
@@ -2666,12 +2667,23 @@ CLASS("Unit", ["Storable" ARG "GOAP_Agent"])
...
 
 	public override METHOD(deserializeFromStorage)
-		params [P_THISOBJECT, P_ARRAY("_serial")];
+		params [P_THISOBJECT, P_ARRAY("_serial"), P_NUMBER("_version")];
 		_serial set [UNIT_DATA_ID_OWNER, 2]; // Server
 		_serial set [UNIT_DATA_ID_MUTEX, MUTEX_NEW()];
 		_serial set [UNIT_DATA_ID_OBJECT_HANDLE, objNull];
 		_serial set [UNIT_DATA_ID_AI, ""];
 
+		// SAVEBREAK >>>
```
Comment (Sparker95) [inline src/Unit/Unit.sqf:2676]: @SomethingSimpl you can see how the versioning works by this example. saveVersion.hpp holds the current save game version value. It was 28, now I set it to 29 when I we add this feature. Here I check if the version of the loaded game is lower than 29, and if so, correct the data upon loading.

**Comment by Sparker95** [comment]
I've moved the text for repair action to stringtable.

Can we perhaps find some building for the repair station instead of this box? I don't like that the box has physics simulation. It means it will clip into something and will kill someone.

**Comment by Sparker95** [comment]
There are several small livonioa buildings called "workshop", I think that they might suit this better. What do you think? Can you experiment with them and pick the one you like?

**Comment by Sparker95** [comment]
I'll merge it now, I've replaced the repair object too. We can replace the repair object later if we like.

**Comment by SomethingSimpl** [comment]
sounds good! thanks for going over it :)

**Pull request merged**